client/blocks/reader-full-post/index.jsx

```diff
@@ -211,6 +211,10 @@ export class FullPostView extends React.Component {
 	}
 
 	parseEmoji() {
+		if ( ! this.refs.article ) {
```

Comment:
There's an attempt to parse emoji on `ReaderFullPostUnavailable` otherwise, which fails because the `ref` is missing:

<img width="1280" alt="screen shot 2016-12-06 at 11 27 37" src="https://cloud.githubusercontent.com/assets/17325/20905398/8073a6b4-bba8-11e6-9f1f-1f2169bf7d16.png">